### PR TITLE
Category is omitted when reading RSS Item.

### PR DESCRIPTION
**Pull request by drago-draganov**
Category is now properly handled when reading RSS Item.
Added support for domain attribute in RSS Category.
Added Tests.

### CODE REVIEW

File: src/Rss/RssParser.cs [hunk 1]
```diff
@@ -331,7 +332,9 @@ public virtual ISyndicationCategory CreateCategory(ISyndicationContent content)
                 throw new FormatException("Invalid Rss category name");
             }
 
-            return new SyndicationCategory(content.Value);
+            return new SyndicationCategory(content.Value) {
+                Scheme = content.Attributes.GetRss(RssConstants.Domain)
```
Comment (jimmyca15) [inline src/Rss/RssParser.cs]: GetRss can return null

Comment (jimmyca15) [reply src/Rss/RssParser.cs]: Just for reference

**Comment by jimmyca15** [comment]
This change will enable categories to be populated when parsing RSS items. In response to #20

**Comment by jimmyca15** [comment]
Correction, fixes #19

File: src/Rss/RssParser.cs (hunk 1 above)
Comment (drago-draganov) [reply src/Rss/RssParser.cs]: It's expected. Scheme can be null

**Pull request merged**